### ISSUE TITLE
Installed from Marketplace doesn't work (v3.0.0)

### ISSUE THREAD

**Issue by pablorq** [scottbedard/oc-debugbar-plugin#31]
Hi,

I've installed debugbar from the Marketplace but it doesn't appear.

I can see it on the stacktrace when an error is logged but it isn't on the frontend nor backend.

Any idea?

Regards

**Comment by pablorq** [comment]
Cache clearing tested, but it isn't working.

```
$ sudo docker exec octobercms_web_1 artisan view:clear
Compiled views cleared!
$ sudo docker exec octobercms_web_1 artisan cache:clear
Cache cleared successfully.
$ sudo docker exec octobercms_web_1 artisan debugbar:clear
Debugbar Storage cleared!
$ sudo docker exec octobercms_web_1 october up
Migrating application and plugins...
System
 - Nothing to migrate.
Backend
 - Nothing to migrate.
Cms
 - Nothing to migrate.
October.Demo
- Nothing to update.
October.Drivers
- Nothing to update.
RainLab.Builder
- Nothing to update.
RainLab.Debugbar
- Nothing to update.
$ 
```

Steps to reproduce:

- Create a new octobercms docker container.
- Install Builder plugin.
-  Install Debugbar plugin.
- No debugbar plugin on frontend nor backend.

> Tested with SQLite and MySQL databases.

Any idea?

**Comment by pablorq** [comment]
I have it working now.

> TL;DR: Add `DEBUGBAR_ENABLED=true` to .env file

Tests done:
- .env file
- Docker environment variable
- Debugbar config file

#### .env file
Add this line to .env file:
```
DEBUGBAR_ENABLED=true
```

>Result: Works!


#### Docker environment variable
Modify or add environment section to docker-compose.yml file
```
environment:
  - DEBUGBAR_ENABLED =true
```
After this run the update to the container:
```
sudo docker-compose up -d
```

>Result: NO debugbar

#### Debugbar config file
Modify the config.php inside the plugin directory (`./html/plugins/rainlab/debugbar/config/config.php`).
From:
```
'enabled' => env('DEBUGBAR_ENABLED', null),
```
To:
```
'enabled' => env('DEBUGBAR_ENABLED', true),
```

>Result: Works!

#### Need verification
According to the documentation, it should be working out of the box if `debug=true` on the app.php config file, but it doesn't.
Please use this as a workaround and do more testing!

**Comment by pablorq** [comment]
Hi!

After 8 months, there's no response about this and today some labels were added and them closed.

Have you tested it? I've tried in different containers and the result is the same.

The main point is why it doesn't work with OctoberCMS debug mode (`debug=true` in app.php), and it has to be forced on the configuration file of DebugBar.

Can you please test that?

Regards

**Comment by bennothommo** [comment]
@pablorq apologies, your last comment said you had found a workaround and it had seemed you were satisfied with that, hence why I closed it.

I have tested this on numerous sites and do not have the same issues you are encountering - I can turn the debug bar on and off simply by changing `debug` to true or false respectively, so I believe there's another step here that is missing between what I am doing on my installs and what you might be doing on your installs.

For example, there should be no need to set `DEBUGBAR_ENABLED` in the environment at all if you want it to be controlled by `app.debug` - the default is to set `DEBUGBAR_ENABLED` to `null` which means it will instead use `app.debug` as expected. I also don't know if it's perhaps Docker doing something tricky.

It's also possible that this has been resolved in a more recent version of the plugin. You should instead use Composer to install the most recent version of October CMS and the Debug Bar plugin and see if that resolves the issue.

**Comment by daftspunk** [comment]
If you need help with this plugin: http://octobercms.com/premium-support
